### ISSUE TITLE
Send the photon color names for gradient for the Primary Hero shelf

### ISSUE THREAD

**Issue by bobsilverberg** [mozilla/addons-server#12132]
In order to avoid having to use inline styles in the front-end for `linear-gradient`, and also to support the use of the `:hover` pseudo-selector for the cta button in the Primary Hero shelf, it would be better if the hero API could provide information that can be used to determine a `className` defined in the front-end.

We discussed that if the hero API returns the names of photon colors as the values for `start` and `end` for `gradient`, then the front-end can concatenate those into a `className` which would include those colors in its definition. This would allow for some synchronization of the colors between  server and front-end without having to manually keep them in sync.

The bottom line of the change is this:

Instead of sending hex colors for  `start` and `end` for `gradient`, send the corresponding photon color names.

**Comment by eviljeff** [comment]
can you provide some sample json for what you'd expect for a given gradient?

**Comment by bobsilverberg** [comment]
> can you provide some sample json for what you'd expect for a given gradient?

It would be pretty much identical to what we get now, but instead of hex color values it would be photon color names. E.g., instead of:

`"gradient":{"start":"#20123A","end":"#582ACB"}`

We would get something like:

`"gradient":{"start":"magenta-50","end":"purple-30"}`

**Comment by bobsilverberg** [comment]
Note that the values we're looking for would correspond to the values in https://github.com/FirefoxUX/photon-colors/blob/master/photon-colors.scss

**Comment by AlexandraMoga** [comment]
Verified on -dev with the `/hero/primary/` endpoint and the gradient start and end values are now sent with the photon color name:

```
 "gradient": {
            "start": "ink-80",
            "end": "blue-70"
        }
```
